canary/workflow_test.go

```diff
@@ -51,7 +52,9 @@ func TestWorkflowTestSuite(t *testing.T) {
 }
 
 func (s *workflowTestSuite) SetupTest() {
+	s.SetLogger(zaptest.NewLogger(s.T()))
```

Comment:
Any reason we are not changing the default [here](https://github.com/uber-go/cadence-client/blob/master/internal/internal_workflow_testsuite.go#L272)?

Reply:
I think that's the right choice, yeah.  Among possibly other locations - any calls to `zap.New...` in pretty much any go code anywhere are immediately suspect IMO.

Reply:
hm.  unfortunately adding it there is somewhere between "not trivial" and "requires breaking changes".  there doesn't appear to be any `*testing.T` available at that point, and if I start changing code to require it, I eventually end up at public APIs.

still very much worthwhile, but tbh I think it's worth tackling separately.